assets/sass/components/_forms.scss

```diff
@@ -482,3 +494,43 @@
         padding-left: $spacingUnit;
     }
 }
+
+.email-signup {
+    background-color: palette('pale-grey');
+    padding: $spacingUnit;
+
+    input[type='submit'] {
+        display: inline-block;
+        vertical-align: middle;
+    }
+
+    .post-submit {
```

Comment:
:neckbeard: Mixing BEM classes makes me twitch a little…but this is fine for now.